content/pt-br/docs/reference/access-authn-authz/authorization.md

```diff
@@ -0,0 +1,248 @@
+
+## Modos de Autorização {#authorization-modules}
+
+O servidor da API Kubernetes pode autorizar uma solicitação usando um dos vários modos de autorização:
+
+ * **Node** - Um modo de autorização de finalidade especial que concede permissões a `kubelets` com base nos `pods` que estão programados para execução. Para saber mais sobre como utilizar o modo de autorização do nó, consulte [Node Authorization](/docs/reference/access-authn-authz/node/).
+ * **ABAC** - Attribute-based access control (ABAC), ou Controle de acesso baseado em atributos, define um paradigma de controle de acesso pelo qual os direitos de acesso são concedidos aos usuários por meio do uso de políticas que combinam atributos. As políticas podem usar qualquer tipo de atributo (atributos de usuário, atributos de recurso, objeto, atributos de ambiente, etc.). Para saber mais sobre como usar o modo ABAC, consulte [ABAC Mode](/docs/reference/access-authn-authz/abac/).
+ * **RBAC** - Role-based access control (RBAC), ou controle de acesso baseado em função, é um método de regular o acesso a recursos de computador ou rede com base nas funções de usuários individuais dentro de uma empresa. Nesse contexto, acesso é a capacidade de um usuário individual realizar uma tarefa específica, como visualizar, criar ou modificar um arquivo. Para saber mais sobre como usar o modo RBAC, consulte [RBAC Mode](/docs/reference/access-authn-authz/rbac/)
+   * Quando especificado RBAC (Role-Based Access Control) usa o group de API `rbac.authorization.k8s.io` para orientar as decisões de autorização, permitindo que os administradores configurem dinamicamente as políticas de permissão por meio da API do Kubernetes.
+   * Para habilitar o modo RBAC, inicie o servidor de API (apiserver) com a opção `--authorization-mode=RBAC`.
+ * **Webhook** - Um WebHook é um retorno de chamada HTTP: um HTTP POST que ocorre quando algo acontece; uma simples notificação de evento via HTTP POST. Um aplicativo da Web que implementa WebHooks postará uma mensagem em um URL quando ocorrerem determinadas coisas. Para saber mais sobre como usar o modo Webhook, consulte [Webhook Mode](/docs/reference/access-authn-authz/webhook/).
+
+#### Verificando acesso a API
+
+`kubectl` fornece o subcomando `auth can-i` para consultar rapidamente a camada de autorização da API.
+O comando usa a API `SelfSubjectAccessReview` para determinar se o usuário atual pode executar
+uma determinada ação e funciona independentemente do modo de autorização utilizado.
+
+
+```bash
+# "can-i create" = "posso criar"
+kubectl auth can-i create deployments --namespace dev
+```
+
+A saída é semelhante a esta:
+
+```
+yes
+```
+
+```shell
+# "can-i create" = "posso criar"
+kubectl auth can-i create deployments --namespace prod
+```
+
+A saída é semelhante a esta:
+
+```
+no
+```
+
+Os administradores podem combinar isso com [user impersonation](/pt-br/docs/reference/access-authn-authz/authentication/#user-impersonation)
```

Comment:
This link is broken because the anchor was not maintained upon translation. The correct anchor would be`#personificação-de-usuário`. You can also use the term "personificação de usuário" in lieu of "user impersonation" to stay consistent with the translated version.